.circleci/documentation-versions.py

```diff
@@ -19,15 +46,24 @@ def main():
     except IndexError:
         sys.exit("Missing build dir path: python /path/docs")
 
-    ignored_dirs = {".git"}
+    # directories which contain docs
     dirs = [
```

Comment:
You could rename this to something like `documentation_directories` and lose the comment above

Reply:
sure no problem